docs/en/tools.md

```diff
@@ -21,16 +21,36 @@ python tools/publish_model.py work_dirs/psenet/latest.pth psenet_r50_fpnf_sbn_1x
 The final output filename will be `psenet_r50_fpnf_sbn_1x_20190801-{hash id}.pth`.
 
 
-## Convert txt annotation to lmdb format
-Sometimes, loading a large txt annotation file with multiple workers can cause OOM (out of memory) error. You can convert the file into lmdb format using `tools/data/utils/txt2lmdb.py` and use LmdbLoader in your config to avoid this issue.
+## Convert text recognition dataset to lmdb format
+Reading images or labels from files can be slow when data is excessive, such as 10 million level. Besides, in the academic community of scene text recognition, most of the datasets are stored in lmdb format, including images and labels. So in order to be more closely integrated with academic and have a more efficient way to store files, we can use `tools/data/utils/lmdb_converter.py` to onvert the existing datasets in mmocr or custom datasets to lmdb format.
+
+| Arguments         | Type       | Description                                                    |
+| ----------------- | ---------- | -------------------------------------------------------------- |
+| `label_path`      | str        | Path to label file.                                            |
+| `output`          | str        | output lmdb path.                                              |
+| `--img-root`      | str        | input imglist path.                                            |
+| `--label-only`    | store_true | Only converter label to lmdb                                   |
+| `--label-format`  | str        | The format of the label file, either txt or jsonl.             |
+| `--batch-size`    | int        | processing batch size, default 1000                            |
+| `--encoding`      | str        | bytes coding scheme, default utf8.                             |
+| `--lmdb_map_size` | int        | maximum size database may grow to , default 109951162776 bytes |
+**Examples**
```

Comment:
```suggestion

### Examples
```

Need a blank line, otherwise these contents will be appended to the table.
Use headings to organize the article so that TOC can be clearly shown in readthedocs